### PR TITLE
clangdev v13.0.0 (redux)

### PR DESCRIPTION
**Pull request by h-vetinari**
Closes #157 

Also add myself to maintainers as discussed [here](https://github.com/conda-forge/clang-compiler-activation-feedstock/pull/69#issuecomment-933036007).

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -51,6 +51,7 @@ requirements:
   host:
     - libcxx {{ cxx_compiler_version }}  # [osx]
     - llvmdev =={{ version }}
+    - zlib                               # [linux]
```
Comment (h-vetinari) [inline recipe/meta.yaml:54]: @isuruf, this is needed by `build.sh` before any of the outputs.

My question is: do we _also_ need to add zlib to all of the outputs individually?

Comment (h-vetinari) [reply recipe/meta.yaml:54]: Currently following a minimal approach to add this only where needed.

**Comment by h-vetinari** [comment]
OK, finally zlib added in enough places. For the python-bindings failure, I have very little to go on, though.

The upstream [readme](https://github.com/llvm/llvm-project/tree/release/13.x/clang/bindings/python) mentions setting `CLANG_LIBRARY_PATH=$(llvm-config --libdir)` (though that's not a recent change, i.e. no change from LLVM 12). Should we set this somewhere in the recipe?

**Comment by isuruf** [comment]
Yes. See https://github.com/conda/conda/issues/10969

**Comment by h-vetinari** [comment]
Ping @isuruf; artefacts are now available.

**Comment by h-vetinari** [comment]
@conda-forge/clangdev

After fixing the libffi-issues on the python-feedstock, this is now green! 🥳 

The only thing that stood out to me from the logs was the following ClobberWarning that we might want to have a look at (possibly in a separate PR to unblock LLVM 13)?

```
ClobberWarning: This transaction has incompatible packages due to a shared path.
  packages: conda-forge/osx-64::clang_bootstrap_osx-64-12.0.1-h3b6de16_2, conda-forge/osx-64::libcxx-12.0.1-habf9029_0
  path: 'include/c++/v1/bitset'
```

**Pull request merged**